matplotcheck/tests/test_polygons.py

```diff
@@ -109,39 +151,10 @@ def test_polygon_custom_fail_message(poly_geo_plot, basic_polygon):
         x, y = basic_polygon.exterior.coords.xy
         poly_list = [(x[0] + 0.5, x[1]) for x in list(zip(x, y))]
         poly_geo_plot.assert_polygons(poly_list, m="Test Message")
-        plt.close()
-
-
-def test_point_geometry_pass(point_geo_plot, pd_gdf):
-    """Check that the point geometry test recognizes correct points."""
-    point_geo_plot.assert_points(points_expected=pd_gdf)
-
-
-def test_point_geometry_fail(point_geo_plot, bad_pd_gdf):
-    """Check that the point geometry test recognizes incorrect points."""
-    with pytest.raises(AssertionError, match="Incorrect Point Data"):
-        point_geo_plot.assert_points(points_expected=bad_pd_gdf)
-
-
-def test_assert_point_fails_list(point_geo_plot, pd_gdf):
-    """
-    Check that the point geometry test fails anything that's not a
-    GeoDataFrame
-    """
-    list_geo = [list(pd_gdf.geometry.x), list(pd_gdf.geometry.y)]
-    with pytest.raises(ValueError, match="points_expected is not expected"):
-        point_geo_plot.assert_points(points_expected=list_geo)
-
-
-def test_get_points(point_geo_plot, pd_gdf):
-    """Tests that get_points returns correct values"""
-    xy_values = point_geo_plot.get_points()
-    assert list(sorted(xy_values.x)) == sorted(list(pd_gdf.geometry.x))
-    assert list(sorted(xy_values.y)) == sorted(list(pd_gdf.geometry.y))
+        plt.close("all")
```

Comment:
just curious -- did we move the point tests somewhere else? just want to ensure i understand this change and it's documented!

Reply:
Yep! They were moved to the `test_points.py` file. If you look all the tests that were removed from here exist over there.

Reply:
ok thank you for the clarification!